django-backend/fecfiler/f3x_summaries/migrations/0009_auto_20220515_2116.py

```diff
@@ -1,10 +1,11 @@
 # Generated by Django 3.2.12 on 2022-05-16 01:16
 
-from fecfiler.f3x_summaries.models import F3XSummary
 from django.db import migrations, models
 
 
 def convert_fecfile_booleans(apps, schema_editor):
+    F3XSummary = apps.get_model("f3x_summaries", "F3XSummary")  # noqa
```

Comment:
The caps in this F3XSummary variable is triggering a code smell in SonarCloud. Is there a convincing argument to keep the code smell and not change the variable name?

Reply:
Updated accordingly.  Coming up with a variable name that communicates what it's doing was a little tricky, but I think I got it.